openverse_catalog/dags/providers/provider_csv_load_scripts/inaturalist/transformed_table.template.sql

```diff
@@ -33,19 +43,14 @@ INSERT INTO {intermediate_table}
             as CREATOR,
         'https://www.inaturalist.org/users/' || INATURALIST.PHOTOS.OBSERVER_ID
             as CREATOR_URL,
-        left(string_agg(INATURALIST.TAXA.NAME, ' & '), 5000) as TITLE,
-        -- TO DO: should there be a timestamp or anything in the metadata or is null ok?
-        null::json as META_DATA,
-        -- TO DO: confirm format here, is provider name integrated, and if so how?
-        array_to_json(string_to_array(string_agg(
-            INATURALIST.TAXA.ancestor_names,
-            '|')
-        ,'|')) as TAGS,
+        taxa_enriched.title,
+        LICENSE_CODES.license_url_metadata as META_DATA,
+        taxa_enriched.tags,
```

Comment:
The tag enrichment is _exactly_ the kind of behavior I was thinking we'd need to have SQL equivalents for 😅 This is great for now, but definitely as we start to have more SQL-only ingestions we'll want some mechanism for abstracting this! Maybe a set of defined functions?

Reply:
Yeah...  I worry about navigating costs of context switching with user defined functions in SQL, particularly for functions that will be run many times over a table, but that might just be something I need to learn to do better in postgres. dbt lets you write jinja macros (e.g. [pivot](https://github.com/dbt-labs/dbt-utils/blob/main/macros/sql/pivot.sql)) for this kind of thing so that you're always running something in SQL, but your code can still be DRY. 

And I guess for this one in particular, I have a bit of a fantasy that we will make a separate `tags` table (e.g. with `media_type, provider, identifier, tag_name` and maybe `authority` if that makes sense) and stop trying to handle them in JSON at all.

But yeah, at a high level, I totally agree that this is not ideal for the longer term. I'm adding it to the list at the top of the PR for now.